src/server/scripts/EasternKingdoms/ScarletEnclave/chapter1.cpp

```diff
@@ -1234,7 +1234,45 @@ class npc_scarlet_miner : public CreatureScript
         }
 };
 
-// npc 28912 quest 17217 boss 29001 mob 29007 go 191092
+enum StableMasterRepo
+{
+    SPELL_REPO = 52265
+};
+
+class spell_stable_master_repo : public SpellScriptLoader
+{
+public:
+    spell_stable_master_repo() : SpellScriptLoader("spell_stable_master_repo") { }
+
+    class spell_stable_master_repo_AuraScript : public AuraScript
+    {
+        PrepareAuraScript(spell_stable_master_repo_AuraScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            return ValidateSpellInfo({ SPELL_REPO });
+        }
+
+        void HandleDummy(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+        {
+            Creature* target = GetTarget()->ToCreature();
```

Comment:
If in any way this spell will be casted on player, worldserver will crash due to this part :P